build.gradle

```diff
@@ -35,6 +36,12 @@ subprojects {
 	apply plugin: 'maven-publish'
 	apply plugin: 'com.jfrog.bintray'
 	apply plugin: 'com.jfrog.artifactory'
+	apply plugin: 'io.freefair.lombok'
+	lombok {
+		version = "1.18.10"
```

Comment:
Would `1.18.16` be compatible? #220 won't work without it

Reply:
Yikes? - on the way